recipes/_install-windows.rb

```diff
@@ -24,17 +24,19 @@
 temp_file = ::File.join(Chef::Config[:file_cache_path], 'ddagent-cli.msi')
 
 # Download the installer to a temp location
-remote_file 'MSI installer' do
-  path temp_file
+remote_file temp_file do
   source node['datadog']['windows_agent_url'] + dd_agent_msi
+  checksum node['datadog']['windows_agent_checksum'] if node['datadog']['windows_agent_checksum']
```

Comment:
One positive to this is also for security, you make sure the file you are downloading is the correct file.  For most of our windows downloads we usually specify the version / checksum which allows for locking down a version however this is more painful when new releases ships from a cookbook management level.  Since it is optional it seemed like a good option to have available for people that want to use.